### ISSUE TITLE
Deterministic trainset broken when valid/test set present

### ISSUE THREAD

**Issue by jboelaert** [activetigger/activetigger#934]
When trying to create a project without randomly sampling the training set, I find that when n_valid or n_test is non-zero the training set is in fact randomly sampled. 

Context: my data has some annotations that I want to import, and is imbalanced on one variable that I want to stratify on. 
In order to make this work, I had to manually shuffle my three data sources, then put the annotated ones on top, then merge them and setting the first (n_train/3) of each source on top, then import without random sampling.

More generally, this raises complex questions about the articulation of the different data importation parameters, with conflicts between: 
- random sampling
- prioritizing
- stratifying
- valid/test

One quick fix to my first problem (non-random sampling + valid/test) could be to use the first n_train rows as training set, then the following n_valid and n_test as validation/test, although this raises the question of the validity of those sets...
Maybe a better solution would be to forbid other options (priority, stratification, valid/test) when importation is non-random?

**Comment by jboelaert** [comment]
I see that this is an old question, as there is a "NEED TO FIX" in the tooltip help of the "Select rows at random" checkbox in project creation form :)

**Comment by AXLMRIN** [comment]
Actually, "NEED TO FIX" comes from our inability to clearly explain what this did when Etienne and I were fixing english typos in the interface 😂
